python-clusters/create-gke-cluster/cluster.json

```diff
@@ -64,15 +64,15 @@
         {
             "name": "podIpRange",
             "label": "Pod IP range",
-            "description": "Use CIDR block/mask notation, e.g. 10.0.0.0/21 or /21",
+            "description": "Range name or CIDR block/mask notation, e.g. 10.0.0.0/21 or /21",
```

Comment:
When allocating a named Secondary range with less than a /21, the error returned by GCP's API is not crystal clear :
```
Requested resource is exhausted: Not all instances running in IGM after 28.906830614s. Expected 2, running 0, transitioning 2. Current errors: [IP_SPACE_EXHAUSTED]: Instance 'gke-jcasoli-gke-test-node-pool-0-9bff35c5-zdvr' creation failed: IP space of 'projects/dataiku-dev/regions/europe-west1/subnetworks/jcasoli-subnet--jcasoli-range-4-58872263a8f31f3e' is exhausted. ."  
```
and lead to a "dirty" situation, with the cluster created but non-functional.
(note that my range name was actually `jcasoli-range-4-24`, and gcp somehow renamed it...)

When directly setting a IP range, the error is clearer (something saying that the prefix is greater than 21), and pops early, so no issues.
So we really at least should add to the description that a block of at least /21 is needed
Bonus point for checking the actual Secondary range, by calling the API.

(btw, setting twice the same secondary range is also rejected by the API, but at least with a meaningful message. Given how trivial it is, maybe we could check it ?)